doc/content/microreactors/gcmr/GCMR_Assembly_Model_Description.md

```diff
@@ -27,6 +27,27 @@ The major technical parameters for the GC-MR assembly model are:
 | Total height (cm) | 200 |
 | Active height (cm) | 160 |
 | TRISO fuel compact radius (cm) | 0.9 |
-| Burnable poison radius (cm) | 0.25 |
-| Coolant hole radius (cm) | 0.6 |
-| Control hole radius (cm) | 0.99 |
+| Moderator compact radius (cm), density ($g/cm^{3}$) | 0.843, 4.2 |
+| Cr coating thickness (cm), density ($g/cm^{3}$) | 0.007, 7.19 |
+| FeCrAl envelope thickness (cm), density ($g/cm^{3}$) | 0.05, 7.055 |
+| Burnable poison compact radius (cm) | 0.25 |
+| Coolant compact radius (cm) | 0.6 |
+| Control compact radius (cm) | 0.99 |
+
+Both the TRISO and B4C poison particles are composed of concentric spherical regions. The particles are packed into graphite compacts. The TRISO particles have the following regions
+
+| Region | Outer Radius (cm) | Density ($g/cm^{3}$) |
+| - | - | - |
+| UCO | 212.5E-4 | 10.5 |
+| C (buffer) | 312.5E-4 | 1.0 |
+| PyC1 | 352.5E-4 | 1.9 |
+| SiC | 387.5E-4 | 3.2 |
+| PyC2 | 427.5E-4 | 1.87 |
+
```

Comment:
densities need updating:
UCO: 10.744
C(buffer): 1.04 (TRISO and BP)
PyC: 1.882 (TRISO and BP)
SiC: 3.171  (TRISO and BP)
B4C: 2.47

Reply:
Glad I posted those to check! I'll fix and push it

Reply:
I confirm I used the same density for this model.